src/MSCorEE.Desktop/MSCorEE.cs

```diff
@@ -72,5 +79,37 @@ public static partial class MSCorEE
         /// </returns>
         [DllImport(nameof(MSCorEE), PreserveSig = true)]
         public static extern unsafe int StrongNameFreeBuffer(byte* pbMemory);
+
+        [DllImport(nameof(MSCorEE), CharSet = CharSet.Unicode, PreserveSig = true)]
+        public static extern int CLRCreateInstance(
+            [In, MarshalAs(UnmanagedType.LPStruct)] Guid rclsid,
+            [In, MarshalAs(UnmanagedType.LPStruct)] Guid riid,
+            [Out, MarshalAs(UnmanagedType.Interface)] out object pMetaHost);
+
+        /// <summary>
+        /// Gets the version number of the common language runtime (CLR) that is associated with the specified process handle. This function has been deprecated in the .NET Framework version 4.
+        /// </summary>
+        /// <param name="hProcess">A handle to a process.</param><param name="buffer">A buffer that contains the version number string upon successful completion of the method.</param><param name="bufferSize">The length of the version buffer.</param><param name="bufferLength">A pointer to the length of the version number string.</param>
+        /// <returns>
+        /// HRESULT
+        /// </returns>
+        /// <remarks>
+        /// .NET Framework Versions: 4.5, 4, 3.5 SP1, 3.5, 3.0 SP1, 3.0, 2.0 SP1, 2.0
+        /// </remarks>
+        [DllImport(nameof(MSCorEE), CharSet = CharSet.Unicode)]
+        public static extern HResult GetVersionFromProcess([In] IntPtr hProcess, [MarshalAs(UnmanagedType.LPWStr), Out] StringBuilder buffer, [In] uint bufferSize, out uint bufferLength);
```

Comment:
Please name parameters to match their header files/docs, per guidelines in contributing.md